pkgs/by-name/wa/watchdogd/package.nix

```
@@ -0,0 +1,24 @@
+{ lib, stdenv, fetchFromGitHub, pkg-config, autoreconfHook, libite, libuev
+, libconfuse }:
```

This is just a suggestion, but I think it would be nice if these arguments were laid out with newlines in between them. It makes it a lot easier to review with a cleaner diff, if anything were to be added or removed.

```suggestion
{ lib
, stdenv
, fetchFromGitHub
, pkg-config
, autoreconfHook
, libite
, libuev
, libconfuse
}:
```

---

I had it this way but nixpkgs-fmt reformatted it again.